pyproject.toml

```diff
@@ -1,13 +1,3 @@
-[tool.black]
-exclude = '''
-(
-    /(
-      | migrations
-    )/
-    | apps.py
-)
-'''
-
```

Comment:
It is a good idea to remove black configurations. We will configure pre-commit for our project. 👍